pkg/util/mount/mount_linux.go

```diff
@@ -240,13 +248,31 @@ func readProcMountsFrom(file io.Reader, out *[]MountPoint) (uint32, error) {
 func (mounter *SafeFormatAndMount) formatAndMount(source string, target string, fstype string, options []string) error {
 	options = append(options, "defaults")
 
+	// Run fsck on the disk to fix repairable issues
+	args := []string{"-a", source}
+	cmd := mounter.Runner.Command("fsck", args...)
```

Comment:
You might want to do something similar to what nsenter_mount.go does to find the absolute path to the fsck binary